### PR TITLE
added bing search support

### PR DESCRIPTION
**Pull request by muskankhedia**
Added the bing search support. #24
A test search is being displayed in the video. To watch the video [click here.](https://drive.google.com/file/d/1z2tdIzDdkshSa3dh9jDCcja1QABRD_2L/view)

### CODE REVIEW

**Comment by Harkishen-Singh** [comment]
Good Work mam!

File: service/controllers/messages.go [hunk 1]
```diff
@@ -115,7 +115,19 @@ func routes(routeObject response, w http.ResponseWriter) {
...
+
+		response := processBingResponses(result)
+		responseJSON := jsonResponseQuery{
+			Status: true,
+			Message: "query-result",
+			Result: response,
+		}
+		jData, _ := json.Marshal(responseJSON)
+		w.Write(jData)
+
```
Comment (Harkishen-Singh) [inline service/controllers/messages.go]: make a call to the Text to Speech in controllers.go, **after you send back the response**.

**Comment by muskankhedia** [comment]
Thank You, Sir.

**Comment by muskankhedia** [comment]
> Good Work mam!

I love you.Sir

**Comment by Harkishen-Singh** [comment]
> > Good Work mam!
> 
> I love you.Sir

love you mam!!

**Comment by muskankhedia** [comment]
> > > Good Work mam!
> > 
> > 
> > I love you.Sir
> 
> love you mam!!

Fuck you.

**Comment by Harkishen-Singh** [comment]
> > > > Good Work mam!
> > > 
> > > 
> > > I love you.Sir
> > 
> > 
> > love you mam!!
> 
> Fuck you.

fuck you !

**Pull request merged**